app/src/main/java/com/example/triptracker/view/map/AddSpot.kt

```diff
@@ -123,279 +187,274 @@ fun AddSpot(recordViewModel: RecordViewModel, latLng: LatLng, onDismiss: () -> U
         }
       }
 
-  // Get the point of interest of the current location if it exists
-  LaunchedEffect(Unit) {
-    recordViewModel.getPOI(latLng)
-    Log.d("POI", recordViewModel.namePOI.value)
-  }
+  Box(
+      modifier =
+          Modifier.fillMaxWidth()
+              .fillMaxHeight(0.85f)
+              .padding(top = 80.dp, start = 15.dp, end = 15.dp, bottom = 20.dp)
+              .background(color = md_theme_light_black, shape = RoundedCornerShape(35.dp))
+              .testTag("AddSpotScreen")) {
+        Column(modifier = Modifier.matchParentSize().verticalScroll(rememberScrollState())) {
+          Row(
+              modifier = Modifier.fillMaxWidth().padding(top = 10.dp),
+              horizontalArrangement = Arrangement.SpaceBetween,
+              verticalAlignment = Alignment.CenterVertically) {
+                Spacer(modifier = Modifier.weight(1f))
+                // Title
+                Text(
+                    text = "Add Spot",
+                    color = Color.White,
```

Comment:
Will the text be displayed in white no matter whether if the phone is in dark mode or not ?

Reply:
Yes since it is on a black background always